### ISSUE TITLE
Wrong training procedure.

### ISSUE THREAD

**Issue by oKatanaaa** [MakiResearchTeam/MakiPoseNet#11]
There are several problems with the training:
1) No background heatmaps are generated.
An example of generation may be seen here:
https://github.com/michalfaber/keras_Realtime_Multi-Person_Pose_Estimation/blob/0f7f5dcd8f2887a9521826dfdcf9abe56997e56a/training/label_maps.py#L7

2) No masks are generated for:
- crowds;
- too close persons;
- unlabeled persons.
These masks must be applied to the loss tensor to avoid penalizing positive predictions.
An example of generation of the masks is available here:
https://github.com/michalfaber/keras_Realtime_Multi-Person_Pose_Estimation/blob/0f7f5dcd8f2887a9521826dfdcf9abe56997e56a/training/dataflow.py#L139

3) No masks are applied to images:
These are the same masks from the point 2.

TO DO:

- [x] Implement generation of heatmaps.
- [x] Implement generation of the masks.
- [x] Implement image augmentation using masks.
